SWLOR.Game.Server/Service/WeatherService.cs

```diff
@@ -218,14 +219,16 @@ private static PlanetaryClimate _GetClimate(NWObject oArea)
                 climate.Heat_Modifier = -5;
                 climate.Humidity_Modifier = -8;
 
+                climate.Snow_Storm = true;
```

Comment:
> Good catch. (Wonder if it even works though....)

I spent some time in here over the weekend. I think there's quite a few things that just don't function - one of them being the thunderstorms. Wouldn't be surprised if there's more to this too, lol.

Reply:
I know when Mith brought this over for us he said he disabled  some of the features. I'm curious if that is what is happening here. It is something I want to look into fixing because having storms and stuff would add a whole new level of depth to module, especially if they can cause detriments to the players that get caught in them. I also want to add DM control to it, but i'm sure this is a lot bigger undertaking then it seems.

Reply:
I spent a couple hours trying to port it over to FF8 and it's pretty confusing. I don't fully understand how the values affect the weather. Definitely more complicated than I had thought. I probably just need to study it a little longer